easybuild/easyconfigs/e/enchant/enchant-1.6.1-intel-2017a.eb

```diff
@@ -0,0 +1,25 @@
+easyblock = 'ConfigureMake'
+
+name = 'enchant'
+version = '1.6.1'
+
+homepage = 'https://abiword.github.io/enchant/'
+description = """Enchant is a library (and command-line program) that wraps a number of different spelling libraries
+ and programs with a consistent interface."""
+
+toolchain = {'name': 'intel', 'version': '2017a'}
+
+source_urls = ['https://github.com/AbiWord/enchant/releases/download/enchant-%s/' % '-'.join(version.split('.'))]
+sources = [SOURCE_TAR_GZ]
+
+builddependencies = [('Autotools', '20150215')]
+dependencies = [('hunspell', '1.6.1')]
```

Comment:
Personally I prefer to have a blank line between the two different types of dependency...